File: java/Weed3/src/noear/weed/cache/LocalCacheRecord.java
package noear.weed.cache;

import java.util.Date;

/**
 * Created by yuety on 14/11/7.
 */
class LocalCacheRecord {
    public Object data;
    public long time;

    public LocalCacheRecord(Object val, int seconds) {
        this.time = new Date().getTime() + seconds * 1000;
        this.data = val;
    }
}

File: .net & mono/Weed3/Cache/LocalCacheRecord.cs
using System;

namespace Noear.Weed.Cache {
    class LocalCacheRecord {
        public int hash;
        public Object data;
        public long time;

        public LocalCacheRecord(int hash, Object val, int seconds) {
            this.hash = hash;
            this.time = new DateTime().AddSeconds(seconds).Ticks;
            this.data = val;
        }
    }
}
